doc_src/tutorial.rst

```diff
@@ -505,13 +505,13 @@ There is also a ``switch`` command::
     end
 
 
-Note that ``case`` does not fall through, and can accept multiple arguments or (quoted) wildcards.
+Note that :ref:`case <cmd-case>` does not fall through, and can accept multiple arguments or (quoted) wildcards.
 
 
 Functions
 ---------
 
-A ``fish`` function is a list of commands, which may optionally take arguments. Unlike other shells, arguments are not passed in "numbered variables" like ``$1``, but instead in a single list ``$argv``. To create a function, use the :ref:`function <cmd-function>` builtin::
+A fish function is a list of commands, which may optionally take arguments. Unlike other shells, arguments are not passed in "numbered variables" like ``$1``, but instead in a single list ``$argv``. To create a function, use the :ref:`function <cmd-function>` builtin::
```

Comment:
Note the fish spelling is #6785. We want to have this consistent, and the consensus seems to be to not add markup unless it's specifically about calling `fish` as a command.

Reply:
Definitely. That's why I changed it.